tests/internal/test_utils_http.py

```diff
@@ -4,7 +4,7 @@
 from ddtrace.internal.utils.http import connector
 
 
-@pytest.mark.parametrize("scheme", ["http", "https", "unix"])
+@pytest.mark.parametrize("scheme", ["http", "https"])
```

Comment:
wait, are we removing support for unix domain sockets?

can/should we do that?

Reply:
We're simply removing a test case. Since we use the common `get_connection`, which is already well-covered in tests, we're now "over-testing" here in fact.

Reply:
well, I would have expected it to still work with `unix://` because it was working before your change.

But I think that is because we do extra checks on `unix://` urls in `get_connection()` which the previous code/this test didn't validate (the port in the url?)

So it is less about over testing, and previously we didn't perform similar validations.

Reply:
> But I think that is because we do extra checks on `unix://` urls in `get_connection()` which the previous code/this test didn't validate (the port in the url?)

That's right, and probably we don't need that kind of support. That's also why I'm suggesting merging behaviour, instead of having something behaving differently.